### DISCUSSION TITLE
树状透视表包含维度下钻功能 如何显示汇总数据

### DISCUSSION THREAD

**Discussion by shockingsrose** [antvis/S2#2377] [Q&A] [answered]
目前树状透视表无法显示汇总数据
![image](https://github.com/antvis/S2/assets/28640478/b5678e9c-be01-45eb-b1ba-725580c90a54)

希望实现官网的这个效果
https://s2.antv.antgroup.com/manual/basic/analysis/drill-down#%E4%BD%BF%E7%94%A8%E5%9C%BA%E6%99%AF

**Comment by wjgogogo** [comment, marked as answer]
树状模式的汇总维度同样需要配合小计配置，才能开启：https://s2.antv.antgroup.com/examples/analysis/totals/#tree
